src/vmainwindow.cpp

```diff
@@ -1065,7 +1065,7 @@ void VMainWindow::initFileMenu()
     fileMenu->addAction(openConfigAct);
 
     QAction *customShortcutAct = new QAction(tr("Custom Shortcuts"), this);
```

Comment:
How about changing this `Custom` to `Customize`, too?

Reply:
That one is fine either way, but I guess `Customize` would be better for the sake of consistency.

Reply:
@Igetin Would you please send an update to this PR? Thanks!

Reply:
@tamlok Done.